Update index.md (#9367)
ethereum/ethereum-org-website: src/content/developers/docs/smart-contracts/deploying/index.md

You should understand [Ethereum networks](/developers/docs/networks/), [transactions](/developers/docs/transactions/) and the [anatomy of smart contracts](/developers/docs/smart-contracts/anatomy/) before deploying smart contracts.

> Deploying a contract also costs ether (ETH) since they are using the storage on the blockchain, so you should be familiar with [gas and fees](/developers/docs/gas/) on Ethereum.

Reviewer: ```suggestion
Deploying a contract also costs ether (ETH) as they are stored on the blockchain, so you should be familiar with [gas and fees](/developers/docs/gas/) on Ethereum.
```

Just small rewording
